core/src/epicli/data/common/defaults/configuration/feature-mapping.yml

```diff
@@ -143,3 +141,7 @@ specification:
       - node-exporter
       - filebeat
       - firewall
+    repository:
```

Comment:
Shouldn't the node exporter and filebeat be here as well?

Reply:
why?

Reply:
to have logs and monitoring enabled on the repository vm

Reply:
done